linode/acceptance/tmpl/linode_cloud_firewall.gotf

```diff
@@ -0,0 +1,38 @@
+{{ define "linode_cloud_firewall" }}
```

Comment:
Can we rename this template to clarify the firewall is used for shielding the testing resources? Like: `e2e_testing_firewall` or `integration_test_firewall` rather than just `linode_cloud_firewall`

Reply:
Yeah `e2e_testing_firewall` seems appropriate